src/elm/Claim.elm

```diff
@@ -389,6 +387,242 @@ viewClaimCard { shared, accountName } profileSummary claim =
+                        div [ class "flex mb-10" ]
+                            [ div [ class "pt-2" ] [ Profile.viewEmpty loggedIn.shared ]
+                            ]
+
+                      else
+                        div [ class "flex -mx-2 pt-2" ]
+                            (List.map
+                                (\c ->
+                                    if c.isApproved then
+                                        div [ class "px-2" ]
+                                            [ Profile.view loggedIn.shared loggedIn.accountName c.validator
+                                            ]
+
+                                    else
+                                        text ""
+                                )
+                                claim.checks
+                            )
+                    ]
+                ]
+
+        disapprovedBy =
+            div
+                [ class claimVerifiersList ]
+                [ p [ class greenTextTitleStyle ] [ text (t "claim.disapproved_by") ]
+                , div [ class "overflow-x-auto" ]
+                    [ if List.filter (\check -> check.isApproved == False) claim.checks |> List.isEmpty then
+                        div [ class "flex mb-10 " ]
+                            [ div [ class "pt-2" ] [ Profile.viewEmpty loggedIn.shared ] ]
+
+                      else
+                        div [ class "flex -mx-2 pt-2" ]
+                            (List.map
+                                (\c ->
+                                    if not c.isApproved then
+                                        div [ class "px-2" ]
+                                            [ Profile.view loggedIn.shared loggedIn.accountName c.validator
+                                            ]
+
+                                    else
+                                        text ""
+                                )
+                                claim.checks
+                            )
+                    ]
+                ]
+
+        pendingVote =
+            let
+                pendingValidators =
+                    List.filter
+                        (\p -> not <| List.member p.account (List.map (\c -> c.validator.account) claim.checks))
+                        claim.action.validators
+            in
+            div
+                [ class claimVerifiersList ]
+                [ p [ class greenTextTitleStyle ] [ text (t "claim.pending_vote") ]
+                , div [ class "overflow-x-auto" ]
+                    [ if List.length claim.checks == claim.action.verifications then
+                        div [ class "flex mb-10 " ]
+                            [ div [ class "pt-2" ] [ Profile.viewEmpty loggedIn.shared ] ]
+
+                      else
+                        div [ class "flex space-x-6" ]
+                            (List.map (\v -> Profile.view loggedIn.shared loggedIn.accountName v) pendingValidators)
```

Comment:
After #537, Profile.view and Profile.viewLarge have been replaced by the functions in Profile.Summary - you need to keep track of a Profile.Summary.Model and have it update on your update function (this is that nested TEA thing you were doing on Neutro!)

This one is trickier because it's actually multiple of them. [Here](https://github.com/cambiatus/frontend/blob/master/src/elm/Page/Dashboard/Analysis.elm#L345-L347) is an example with multiple of them being handled